epub33/rs/index.html

```diff
@@ -1818,11 +1823,21 @@ <h4>Event Model Considerations</h4>
 			</section>
 		</section>
 		<section id="app-epubReadingSystem" class="appendix">
-			<h2>JavaScript <dfn>epubReadingSystem</dfn> Object</h2>
+			<h2><dfn>epubReadingSystem</dfn> Object</h2>
+
+			<p class="issue atrisk" title="Are there enough implementations?">Although this section is
+					<em>normative</em>, it is not clear whether implementations of this interface are widespread enough
+				to warrant this status. If it turns out not to be the case, this section will be marked as
+					<em>non-normative</em> in the final version of the specification.</p>
 
 			<section id="app-ers-idl">
 				<h3>Interface Definition</h3>
 
+				<p> This interface definition uses the [[!WEBIDL]] notation, because it is aimed Reading System that,
+					conceptually, act as the core rendering engines of Web browsers providing a scripting environment
```

Comment:
Oddly phrased. Maybe something more like "act as the core rendering engines of epub content in Web browsers"? Still a little clumsy.

Reply:
This also looks like a note. I've made it one in the last commit and tried to rephrase.